docs/content/en/docs/operator-manual/piped/installation.md

```diff
@@ -93,7 +107,58 @@ Most part is identical to the previous way, but some part is slightly different.
     --set rbac.scope=namespace
   ```
 
-### Installing on single machine
+### Install on OpenShift less than 4.2
```

Comment:
Not so sure but maybe add a note to this header such as `Install on OpenShift less than 4.2 (special/edge case)` or make this as a subheader of __In the namespaced mode__ above is a better option 🤔

Reply:
for sure. let me give it a try to employ the subheader pattern!